### PR TITLE
Update Roslyn dependency version

### PR DESCRIPTION
**Pull request by sebastien-marichal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +54,16 @@ The full list of changes is contained is available on the [release page](https:/
 * v2.0 is built against Roslyn 2.8.2, so will work against analyzers that use that version of Roslyn or earlier.
 * v2.0 uses NuGet v4.7, which supports the TLS1.3 security protocol.
 
+#### Updating Roslyn
+
+The SDK is compatible with analyzer targeting Roslyn from version 1.0.0 up to the version specified in [Directory.Build.props](https://github.com/SonarSource/sonarqube-roslyn-sdk/blob/master/Directory.Build.props#L9).
```
Comment (pavel-mikula-sonarsource) [inline README.md:59]: Let's make this path relative. If it moves, it will still work

File: README.md [hunk 2]
```diff
@@ -125,6 +132,8 @@ parameter. This will overwrite the above mentioned NuGet behaviour.
 #### Generating a jar for an analyzer that is not available from a NuGet feed
 If you want to create a jar for Roslyn analyzer that is not available from a NuGet feed (e.g. an analyzer you have created on your local machine) you can specify a package source that points at a local directory containing the *.nupkg* file created by the standard Roslyn templates. See the [NuGet docs](https://docs.nuget.org/create/hosting-your-own-nuget-feeds) for more information.
 
+By default, the [NuGet.config](https://github.com/SonarSource/sonarqube-roslyn-sdk/blob/master/RoslynPluginGenerator/NuGet.config#L16) file shipped with the RoslynSonarQubeGenerator has a local package source configured that points to `C:\LocalNugetFeed`.
```
Comment (pavel-mikula-sonarsource) [inline README.md:135]: Same, let's make the URL relative

File: README.md (hunk 1 above)
Comment (pavel-mikula-sonarsource) [inline README.md:59]: There something wrong, when opening the file itself, the link does not work

**Review by pavel-mikula-sonarsource — CHANGES_REQUESTED** [review]
Link polishing

File: README.md (hunk 1 above)
Comment (sebastien-marichal) [reply README.md]: I don't see any issue when opening the link.
What is the issue on your side?

Comment (pavel-mikula-sonarsource) [reply README.md]: Works now

**Review by pavel-mikula-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**